src/shaders/_prelude.vertex.glsl

```diff
@@ -16,6 +16,37 @@ precision highp float;
 
 #endif
 
+#ifndef EXTENT
+#define EXTENT 8192.0
```

Comment:
Do we have to define this here if it's already defined in `_prelude.glsl` ? As I see it should be included in both the vertex and fragment shaders already.

Reply:
Good call, fixed that. The order in which the preludes were injected was incorrect (_prelude.glsl was coming after _prelude.vertex.glsl and _prelude.fragment.glsl). Refer https://github.com/mapbox/mapbox-gl-js/pull/11015/commits/d86bc5e19d69dd66f266b78b1b222a4f9ed9d969.